app/src/org/commcare/activities/FormRecordListActivity.java

```diff
@@ -557,19 +557,21 @@ public boolean onQueryTextChange(String newText) {
     public boolean onPrepareOptionsMenu(Menu menu) {
         super.onPrepareOptionsMenu(menu);
 
-        MenuItem quarantine = menu.findItem(MENU_SUBMIT_QUARANTINE_REPORT);
-        if (quarantine != null) {
-            quarantine.setVisible(FormRecordFilter.Limbo.equals(adapter.getFilter()));
-        }
+        if (adapter != null) {
```

Comment:
diff is not super clear though the change is just that this null check is now wrapping the rest of the code in `onPrepareOptionsMenu`